notebooks/multimodal_vdj/01_preprocessing.ipynb

```diff
@@ -0,0 +1,3169 @@
+{
```

Comment:
<p>So what is the goal of the quality control chapter What do we want to get rid of? Doublets, yes. More things? Might be worth mentioning here. Also, what are the consequences of trash data with doublets? Might add that here as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>